src/components/Admin/utils/DataController.js

```diff
@@ -0,0 +1,114 @@
+import firebase from 'firebase/app';
+import 'firebase/firestore';
+import 'firebase/storage';
+import 'firebase/auth';
+import getStore from '../../../utils/honestyStore';
+
+class DataController {
+  constructor() {
+    this.storage = firebase.storage().ref();
+    this.db = firebase.firestore();
+    window.db = this.db;
```

Comment:
you make a variable and then reassign its name? is this what you are suppose to do?

Reply:
Oops didn't mean to commit that, debugging.